src/project-css/app/Http/Controllers/UploadController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Storage;
+use App\Collection;
+use App\User;
+use App\Table;
+use Auth;
+
+class UploadController extends Controller{
```

Comment:
You should think about building validation methods to work on mitigating the number of conditions.